src/signTx.c

```diff
@@ -286,26 +323,52 @@ static void signTx_handleInitAPDU(uint8_t p2, uint8_t* wireDataBuffer, size_t wi
 		ctx->commonTxData.protocolMagic = u4be_read(wireHeader->protocolMagic);
 		TRACE("protocol magic %d", ctx->commonTxData.protocolMagic);
 
+		switch (wireHeader->includeTtl) {
+		case SIGN_TX_INCLUDED_YES:
```

Comment:
may be worth factoring out the logic to parse the YES/NO enum into a boolean value, given that it's repeated three times already